Sources/Swift/Integrations/SessionReplay/SentryBreadcrumbReplayConverter.swift

```diff
@@ -0,0 +1,92 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryBreadcrumbReplayConverter: NSObject {
+    
+    private let supportedNetworkData = Set<String>([
+        "status_code",
+        "method",
+        "response_content_length",
+        "request_content_length",
+        "http.query",
+        "http.fragment"]
+    )
+    
+    func replayBreadcrumbs(_ breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent] {
```

Comment:
also nit: I'd expect the main method of the `XXXConverter` class to be just `convert` 🙂